### DISCUSSION TITLE
what if dataset and citation paper have different publication dates?

### DISCUSSION THREAD

**Discussion by lucasrodes** [owid/etl#1665] [Metadata] [answered]
Sometimes, I find a data product with a publication date different from that of the paper accompanying it.

See [this example](https://github.com/owid/etl/pull/1658/files#diff-5cdd1d9eb50e67f9ccc12397206a519b3cbb8ee00485e9cb89a49f71bf89d04b), where the publication of the paper is from 1999, but the data has been updated a posterior and the latest update was in 2007.

How should we set `date_published` in these cases?

**Comment by pabloarosado** [comment, marked as answer]
I think in most cases, including this, the most relevant publication date is the date of the latest update of the data product. That's "the date of the data". On the other hand, the publication date of the paper is "the date of the methodology".
In general, we need to provide the information that is most useful to the user. The metadata we write on the snapshot step is mostly referring to a "data product". In some cases, the data product may be a file that gets updated regularly, and whose documentation is an old paper. In such cases, we don't want to document the paper: We want to document the data product.
